model_compression_toolkit/qat/pytorch/quantizer/quantization_builder.py

```diff
@@ -29,14 +29,15 @@
     BasePytorchActivationTrainableQuantizer
 
 
-def get_activation_quantizer_holder(n: common.BaseNode,
+def get_activation_quantizer_holder(n: common.BaseNode, prev_node: common.BaseNode,
```

Comment:
Same here, it is preferred to find a way to implement the new capability (preserving quantization) without needing to change unrelated functions.